translations/monero-core_es.ts

```diff
@@ -524,12 +524,12 @@
     <message>
         <location filename="../LeftPanel.qml" line="509"/>
         <source>Y</source>
-        <translation type="unfinished"></translation>
+        <translation>Y</translation>
     </message>
     <message>
         <location filename="../LeftPanel.qml" line="463"/>
         <source>Sign/verify</source>
-        <translation>Firmar/verificar</translation>
+        <translation>Firmar o verificar</translation>
```

Comment:
Firmar/verificar

Puntuaciones deben ser iguales, Hay que seguir [las guias :)](https://taiga.getmonero.org/project/erciccione-monero-localization/wiki/translating-the-gui)